frontend/README.md

```diff
@@ -1,20 +1,15 @@
 Tobira Frontend
-==============
+===============
 
 
 Building the Tobira Frontend
 ---------------------------
 
-Use NPM to get all necessary dependencies and to build the Tobira frontend:
+Use npm to build Tobira's frontend:
+This requires the backend-generated GraphQL schema at `src/schema.graphql`.
 
 ```sh
 npm ci
 npx relay-compiler
 npm run build:dev
 ```
```

Comment:
Should we mention the other NPM tasks here? Linting? Type-checking? I guess the question is: Is this a guide to build this once for a deployment, or is it for developers, as well?

Reply:
Above, you already argued for short docs. My intention (for now) is to tell people how to get started.
I'm not opposed to adding more stuff here later _if_ it's generally important and it's not explaining default tooling.